modules/juce_audio_formats/codecs/juce_WindowsMediaAudioFormat.cpp

```diff
@@ -22,6 +22,304 @@
   ==============================================================================
 */
 
+#if JUCE_MINGW
+
+} // close juce namespace
+
+#include <guiddef.h>
+#include <tuple>
+
+using	tupluuid = std::tuple<uint32_t, uint16_t, uint16_t, uint8_t, uint8_t, uint8_t, uint8_t, uint8_t, uint8_t, uint8_t, uint8_t>;
+
+inline
+tupluuid	UUID_split(const char *s)
+{	
+	GUID	g = juce::uuidFromString(s);
+	
+	return std::make_tuple(g.Data1, g.Data2, g.Data3, g.Data4[0], g.Data4[1], g.Data4[2], g.Data4[3], g.Data4[4], g.Data4[5], g.Data4[6], g.Data4[7]);
+	// __CRT_UUID_DECL(name, g.Data1, g.Data2, g.Data3, g.Data4[0], g.Data4[1], g.Data4[2], g.Data4[3], g.Data4[4], g.Data4[5], g.Data4[6], g.Data4[7]);
+}
+
+/*
+struct bite
+{
+	bite(tupluuid tup)
+		: m_tupid(tup)
+	{
+	}
+	
+	auto&	operator[](const size_t idx)
+	{	return std::get<idx>(m_tupid);
+	}
+	
+	tupluuid	m_tupid;
+};
+*/
+
+/*
+#define	gh(s)	gx = UUID_split(s);
+#define	DeclUUID(typ, s)	__CRT_UUID_DECL(typ,	get<0>(gx), get<1>(gx), get<2>(gx), get<3>(gx), get<4>(gx), \
+							get<5>(gx), get<6>(gx), get<7>(gx), get<8>(gx), get<9>(gx), get<10>(gx));
+*/
+
+typedef 
+enum WMT_RIGHTS
+{
+	WMT_RIGHT_PLAYBACK			= 0x1,
+	WMT_RIGHT_COPY_TO_NON_SDMI_DEVICE	= 0x2,
+	WMT_RIGHT_COPY_TO_CD			= 0x8,
+	WMT_RIGHT_COPY_TO_SDMI_DEVICE		= 0x10,
+	WMT_RIGHT_ONE_TIME			= 0x20,
+	WMT_RIGHT_SAVE_STREAM_PROTECTED		= 0x40,
+	WMT_RIGHT_COPY				= 0x80,
+	WMT_RIGHT_COLLABORATIVE_PLAY		= 0x100,
+	WMT_RIGHT_SDMI_TRIGGER			= 0x10000,
+	WMT_RIGHT_SDMI_NOMORECOPIES		= 0x20000
+} WMT_RIGHTS;
+    
+#include "C:/Program Files (x86)/Windows Kits/8.1/Include/um/nserror.h"

```

Comment:
well that's embarrassing :/ no sure why that was needed, possibly one of the MS files that mingw couldn't legally bundle but that might have changed by now. Let me know if/how I can help.